client/src/components/Film.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import {FilmDescriptionItem} from './FilmDescriptionItem.js';
+import {FilmPoster} from './FilmPoster.js';
+
+export default class Film extends React.Component {
+  render() {
+    return (
+      <div className="media-show">
+        <div className="media-show__wrapper">
+          <FilmPoster poster={this.props.poster} show_title={this.props.show_title}/>
```

Comment:
all properties should be in camelCase. 'show_title' should be 'showTitle'.
I think your eslint doesn't work. Try to understand how it work, and install it to your texeditor or IDE whitch you are using for writing this code.

You should use all convetion rules in code which containt here: https://github.com/airbnb/javascript/tree/master/react#props
Not only props.and also all conventions whitch are writed in this file.
And also all convetions whitch are writed here : https://github.com/airbnb/javascript